docker/docker.go

```diff
@@ -217,23 +217,23 @@ func IsContainerRunning(ctx context.Context, dClient *client.Client, hostname st
 	return false, nil
 }
 
-func localImageExists(ctx context.Context, dClient *client.Client, hostname string, containerImage string) (bool, error) {
+func localImageExists(ctx context.Context, dClient *client.Client, hostname string, containerImage string) error {
 	var err error
 	for i := 1; i <= RetryCount; i++ {
 		logrus.Infof("Checking if image [%s] exists on host [%s], try #%d", containerImage, hostname, i)
 		_, _, err = dClient.ImageInspectWithRaw(ctx, containerImage)
 		if err != nil {
 			if client.IsErrNotFound(err) {
-				logrus.Infof("Image [%s] does not exist on host [%s]: %v", containerImage, hostname, err)
-				return false, nil
+				logrus.Debugf("Image [%s] does not exist on host [%s]: %v", containerImage, hostname, err)
+				return err
```

Comment:
This is not an error situation, returning false, nil is correct as the function is `localImageExists` and the error from the Docker daemon is not found, meaning it does not exist and should return false.